### PR TITLE
remove deprecated errorMessage property

### LINKED ISSUE #8275: Replace errorMessage with message
The `errorMessage` property from the `ISummaryNack` interface has been deprecated and will be removed from `common\lib\protocol-definitions\src\protocol.ts`.

All uses of `errorMessage` must be replaced with `message`.

### PR DESCRIPTION
**Pull request by scottn12**
The `errorMessage` property from the `ISummaryNack` interface has been deprecated and will be removed from `common\lib\protocol-definitions\src\protocol.ts`.

All uses of `errorMessage` must be replaced with `message`.

Closes #8275

### CODE REVIEW

File: server/routerlicious/packages/lambdas/src/scribe/lambda.ts [hunk 1]
```diff
@@ -236,7 +236,7 @@ export class ScribeLambda implements IPartitionLambda {
                                     const nackMessage = summaryResponse.message as ISummaryNack;
                                     await this.sendSummaryNack(nackMessage);
                                     const errorMsg = `Client summary failure @${value.operation.sequenceNumber}. `
-                                        + `Error: ${nackMessage.message ?? nackMessage.errorMessage}`;
+                                        + `Error: ${nackMessage.message ?? nackMessage.message}`;
```
Comment (sdeshpande3) [inline server/routerlicious/packages/lambdas/src/scribe/lambda.ts:239]: seems like a duplicate here
```ts
`Error: ${nackMessage.message}`;
```

File: packages/runtime/container-runtime/src/summaryGenerator.ts [hunk 2]
```diff
@@ -349,7 +349,7 @@ export class SummaryGenerator {
                 // back-compat: cast needed until dep on protocol-definitions version bump
```
Comment (heliocliu) [inline packages/runtime/container-runtime/src/summaryGenerator.ts:349]: can you check the history for this comment for how long ago it was?  the code below on L351 is handling the back-compat stuff already so if this is really safe to remove this whole block should be cleaned up

Comment (vladsud) [inline packages/runtime/container-runtime/src/summaryGenerator.ts:349]: Can you please check if this comment is still valid (or we can actually remove cast below?)

**Comment by vladsud** [comment]
Please validate (in debugger) that indeed we get proper message, not errorMessage (for SPO & r11s)

**Review by vladsud — APPROVED** [review]

**Pull request merged**